### PR TITLE
Add custom vspec handling from digital auto

### PR DESCRIPTION
**Pull request by dennismeister93**
## Describe your changes
Add custom vspec handling from digital auto
## Issue ticket number and link
[#12952](https://softwaredefinedvehicle.visualstudio.com/SoftwareDefinedVehicle/_workitems/edit/12952)
## Checklist before requesting a review
- [x] I have performed a self-review of my code
- [ ] If it is a core feature, I have added thorough tests.
- [ ] Do we need to implement analytics?
- [ ] Will this be part of a product update? If yes, please write one phrase about this update.

### CODE REVIEW

File: content/en/docs/tutorials/prototyping/_index.md [hunk 1]
```diff
@@ -21,10 +21,14 @@ As first step open [playground.digital.auto](https://digitalauto.netlify.app/),
 ![playground.digital.auto](./digital-auto.png)
 ![vehicle-models](./vehicle-models.png)
 
-You now have the option to browse existing vehicle signals for the selected vehicle model which you can use for prototyping your _Vehicle App_ by clicking on _Vehicle APIs_.
+You now have the possibility to browse existing vehicle signals for the selected vehicle model which you can use for prototyping your _Vehicle App_ by clicking on _Vehicle APIs_.
 
 ![cvi-catalogue](./cvi-catalogue.png)
 
+If the ideation of your vehicle app prototype comes with any new Vehicle API which is not part of the standard [VSS](https://covesa.github.io/vehicle_signal_specification/introduction/) you also have the option to include it into your preselected model by clicking the _+ New Wishlist API_ button (You need an account for that, please contact us for getting one). After adding a name, description, type, datatype and clicking on _create_ the new API is ready to be used in your prototype.
```
Comment (MP91) [inline content/en/docs/tutorials/prototyping/_index.md:28]: you need an account and must be contributor. please add this condition

File: content/en/docs/tutorials/prototyping/_index.md [hunk 2]
```diff
@@ -35,23 +39,23 @@ The next step would be to prototype your idea. To do so:
...
+If you press this button you will be forwarded to [GitHub](https://github.com/) to login with your GitHub Account and authorize [velocitas-project-generator](https://github.com/eclipse-velocitas/velocitas-project-generator-npm) to create the repository for you. After you authorized the project generator you will be redirected to the [playground.digital.auto](https://digitalauto.netlify.app/) and asked for a repository name (Which also is the app's name). After pressing "Create repository" the project generator takes over your prototype code, adapts it to the structure in the [vehicle-app-python-template](https://github.com/eclipse-velocitas/vehicle-app-python-template) and creates a new private repository under your specified GitHub User/Organisation.
```
Comment (MP91) [inline content/en/docs/tutorials/prototyping/_index.md:58]: The organisation can not yet be specified or?

File: content/en/docs/tutorials/prototyping/_index.md [hunk 3]
```diff
@@ -60,6 +64,7 @@ After the generation of the repository is completed a pop-up dialogue with the U
 - _/app/requirements.txt_ with definition of dependencies
 - _/.devcontainer_/ required scripts to install every prerequisite in Microsoft Visual Studio Code
 - _/.github/workflows/_ with all required CI/CD pipelines to build, test and deploy the vehicle application as container image to the GitHub container registry
+- if your prototype includes any exceptional API you added beforehand our [automated vehicle model lifecycle](/docs/tutorials/vehicle_model_creation/automated_model_lifecycle) takes care of handling the custom VSS vspec file coming from digital.auto.
```
Comment (MP91) [inline content/en/docs/tutorials/prototyping/_index.md:67]: please add the _/gen/ folder here

File: content/en/docs/tutorials/prototyping/run-section.png
Comment (MP91) [inline content/en/docs/tutorials/prototyping/run-section.png:1]: all images are now larger (mostly higher). is this wanted?

**Review by MP91 — CHANGES_REQUESTED** [review]

File: content/en/docs/tutorials/prototyping/code-section.png
Comment (doosuu) [inline content/en/docs/tutorials/prototyping/code-section.png:1]: Add visual hints to the element of the website you want to want to emphasize.

File: content/en/docs/tutorials/prototyping/cvi-catalogue.png
Comment (doosuu) [inline content/en/docs/tutorials/prototyping/cvi-catalogue.png:1]: Add visual hints to the element of the website you want to want to emphasize.

File: content/en/docs/tutorials/prototyping/generate.png
Comment (doosuu) [inline content/en/docs/tutorials/prototyping/generate.png:1]: Add visual hints to the element of the website you want to want to emphasize.

File: content/en/docs/tutorials/prototyping/run-section.png
Comment (doosuu) [inline content/en/docs/tutorials/prototyping/run-section.png:1]: Add visual hints to the element of the website you want to want to emphasize.

File: content/en/docs/tutorials/prototyping/wishlist.png
Comment (doosuu) [inline content/en/docs/tutorials/prototyping/wishlist.png:1]: Add visual hints to the element of the website you want to want to emphasize.

**Review by doosuu — CHANGES_REQUESTED** [review]

File: content/en/docs/tutorials/prototyping/_index.md [hunk 4]
```diff
@@ -6,62 +6,112 @@ description: >
...
+
+Among other things the newly created repository will contain:
+
+{{<table "table table-bordered">}}
+| Files | Description |
+|-------|-------------|
+|_/app/src/main.py_| containing your modified prototype code |
+|_/app/AppManifest.json_| with definition of required services |
+|_/app/requirements.txt_| with definition of dependencies |
+|_/.devcontainer/_| required scripts to install every prerequisite in Microsoft Visual Studio Code |
```
Comment (MP91) [inline content/en/docs/tutorials/prototyping/_index.md:105]:
```suggestion
|_/.devcontainer/_| Required scripts and settings to setup the devcontainer in Microsoft Visual Studio Code |
```

File: content/en/docs/tutorials/prototyping/_index.md [hunk 5]
```diff
@@ -6,62 +6,112 @@ description: >
...
+A succesful generation of the repository is followed by a pop-up dialogue with the URL of your repository. 
+
+Among other things the newly created repository will contain:
+
+{{<table "table table-bordered">}}
+| Files | Description |
+|-------|-------------|
+|_/app/src/main.py_| containing your modified prototype code |
+|_/app/AppManifest.json_| with definition of required services |
+|_/app/requirements.txt_| with definition of dependencies |
```
Comment (MP91) [inline content/en/docs/tutorials/prototyping/_index.md:104]:
```suggestion
|_/app/AppManifest.json_| Settings file defining required services |
|_/app/requirements.txt_| Requirements file defining all python dependencies |
```

File: content/en/docs/tutorials/prototyping/_index.md [hunk 6]
```diff
@@ -6,62 +6,112 @@ description: >
...
+|_/.devcontainer/_| required scripts to install every prerequisite in Microsoft Visual Studio Code |
+|_/.github/workflows/_| with all required CI/CD pipelines to build, test and deploy the vehicle application as container image to the GitHub container registry|
```
Comment (MP91) [inline content/en/docs/tutorials/prototyping/_index.md:106]:
```suggestion
|_/.github/workflows/_| All required CI/CD pipelines to build, test and deploy the vehicle application as container image to the GitHub container registry|
```

File: content/en/docs/tutorials/prototyping/_index.md [hunk 7]
```diff
@@ -6,62 +6,112 @@ description: >
...
+|_/gen/vehicle\_model/_| if your prototype includes any exceptional API you added beforehand our [automated vehicle model lifecycle](/docs/tutorials/vehicle_model_creation/automated_model_lifecycle) takes care of handling the custom VSS vspec file coming from digital.auto and generates a vehicle_model when starting the devContainer |
```
Comment (MP91) [inline content/en/docs/tutorials/prototyping/_index.md:107]:
```suggestion
|_/gen/vehicle\_model/_| The generated model classes. If your prototype includes any exceptional API you added beforehand our [automated vehicle model lifecycle](/docs/tutorials/vehicle_model_creation/automated_model_lifecycle) takes care of handling the custom VSS vspec file coming from digital.auto and generates a vehicle_model when starting the devContainer |
```

**Review by MP91 — CHANGES_REQUESTED** [review]

**Review by BjoernAtBosch — APPROVED** [review]
LGTM

**Review by MP91 — APPROVED** [review]
LGTM

**Review by doosuu — APPROVED** [review]
LGTM

**Pull request merged**